index.html

```diff
@@ -6,5 +6,7 @@
   </head>
```

Comment:
Don't forget to add your name to the `<title></title>` section

Reply:
Done!